src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -431,6 +431,14 @@ internal override bool IsDeclaredReadOnly
                     return true;
                 }
 
+                // can this accessor be made implicitly readonly?
+
+                var options = (CSharpParseOptions)SyntaxTree.Options;
+                if (!options.IsFeatureEnabled(MessageID.IDS_FeatureReadOnlyMembers))
```

Comment:
I believe @jaredpar made a comment that this is okay to do for 7.2 and above (but not below), due to the PEVerify requirement (https://github.com/dotnet/roslyn/issues/35113#issuecomment-485265659)....

This is just disabling it in general unless you are on 8.0 though, correct?

Reply:
If it was for 7.2 and above, it would be gated on the `PEVerify compat` flag (I forget its actual name) instead, right?

Reply:
I think it's simpler if it's 8.0  as this is when the feature was introduced. I think conceptually it could be 7.2 and meet our PEVerify requirements but I think that's getting too cute. Keepit simple and bind it to the feature that itroduced it. 

---
In reply to: [277822085](https://github.com/dotnet/roslyn/pull/35213#discussion_r277822085) [](ancestors = 277822085)